README.md

```diff
@@ -1,12 +1,11 @@
 <h1 align="center">
-  Free Robotics web page
+  air4childrem web page
 </h1>
 
 The main intention of this repository is to provide relevant information of the project `Free Robotics`.
 
 ## Getting started
-This is a Gatsby.js web development. In order to run this code you have to install the following: 
-
+This is a Gatsby.js web development. In order to run this code you have type the following commands in console:
 
```

Comment:
Probably prerequisites section can be added such as the OS distro version in the machine

Reply:
Check this [commit](https://github.com/air4children/site/pull/5/commits/4dca893d99a4acf8f99525366828e8c30a65ce32). I added a more descriptive information.